benchmark/benchmark-tidb-using-tpcc.md

```diff
@@ -70,23 +70,49 @@ ant
 | RAM | 128GB |
 | DISK | Optane 500GB SSD |
 
-因为该型号 CPU 是 NUMA 架构，建议先用 `taskset` 进行绑核，首先用 `lscpu` 查看 NUMA node，比如：
+因为该型号 CPU 是 NUMA 架构，建议用 `numactl` 进行绑核。
 
-```text
-NUMA node0 CPU(s):     0,2,4,6,8,10,12,14,16,18,20,22,24,26,28,30,32,34,36,38
-NUMA node1 CPU(s):     1,3,5,7,9,11,13,15,17,19,21,23,25,27,29,31,33,35,37,39
-```
+1. [安装 numactl 工具](/check-before-deployment.md#安装-numactl-工具) 。
```

Comment:
```suggestion
1. [安装 numactl 工具](https://docs.pingcap.com/zh/tidb/v4.0/check-before-deployment#安装-numactl-工具) 。
```
The link is dead. I replace it with an absolute link. PTAL.

Reply:
LGTM